### PR TITLE
docs: incentives for protocol devs

### PR DESCRIPTION
**Pull request by H4NLee**
For now, our incentives (or reward distribution) is quite simple, so I just write down simply.
If you think some more contents are needed in this doc, please leave comments.

FYI, I intentionally did not write down the contents related to the oracle commission rate(plus max rate and max change rate). Because it seems more related to oracle.

### CODE REVIEW

**Review by audtlr24 — APPROVED** [review]
lgtm. For the consumer's veiw, how about writing down the details of depositing funds when making a deal and refunding them when deactivating? 

I'm not sure if this would be better put into `2-data-deal.md`.

**Comment by H4NLee** [comment]
> lgtm. For the consumer's veiw, how about writing down the details of depositing funds when making a deal and refunding them when deactivating?
I'm not sure if this would be better put into 2-data-deal.md.

It is written in `2-data-deal.md` briefly, but I agree that it would be great to add as you said.
It can provide the whole token flow of DEP. I'll add it.

File: .gitbook/3-protocol-devs/1-dep-specs/6-incentives.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+## Technical Specification
+
+### Reward Distribution
+
+When [creating a data deal](./2-data-deal.md#create-data-deal), data consumers specify the quantity and the budget for the data.
+From this, the price per data can be calculated as:
+
+```
+price_per_data = deposit / max_num_data
+```
```
Comment (inchori) [inline .gitbook/3-protocol-devs/1-dep-specs/6-incentives.md:35]: I think that it will be good for explaining how does the remainder calculated.

**Review by inchori — APPROVED** [review]
lgtm

**Review by gyuguen — APPROVED** [review]
LGTM

File: .gitbook/3-protocol-devs/1-dep-specs/6-incentives.md (hunk 1 above)
Comment (H4NLee) [reply .gitbook/3-protocol-devs/1-dep-specs/6-incentives.md:35]: Actually, we don't calculate remaining budget.
When consumer deactivate the deal, just balance goes back to consumer (by using `bankkeeper.GetBalance`).

**Review by youngjoon-lee — APPROVED** [review]
Good enough!

**Review by audtlr24 — APPROVED** [review]
lgtm

**Review by inchori — APPROVED** [review]
lgtm

**Pull request merged**